src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int ZERO_INDEX = 0;
+    public static final int FIRST_INDEX = 1;
+    public static final int AMOUNTS_INDEX = 2;
+    public static final String SPLIT_PATTERN = "\\W";
+    public static final String SEPARATOR = System.lineSeparator();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(readAndCount(fromFileName));
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName, true))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can’t write to file");
+        }
```

Comment:
what about creating separate method for writing into the file and catching the exception there?

Reply:
Wrote in slack in Ukrainian, check please